uint32 tow   # Milliseconds since start of GPS week. If the high bit is set, the time is unknown or invalid.
uint8  tow_f # Milliseconds since start of GPS week, fractional part
